index.html

```diff
@@ -93,6 +93,7 @@
                                     <li class="social-link"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
                                     <li class="social-link"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
                                     <li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+				    <li class="social-link"><a target="_self" href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a></li>
```

Comment:
Add a single space to intend this line

Reply:
it is already indented; its just showing like that - wait ill send an image :D

Reply:
![screen shot 2018-01-02 at 9 50 25 am](https://user-images.githubusercontent.com/33036913/34474247-8a0f230e-efa2-11e7-8f83-f69c2fea9f03.png)

Reply:
Look the line 96 where i have made the changes; it is indented properly :D